### ISSUE TITLE
[Feature Request] Add button to clear current playlist

### ISSUE THREAD

**Issue by Dreadeath** [ludouzi/fooyin#114]
Please add button to clear current playlist to playlist control buttons.

**Comment by ludouzi** [comment]
There's already Edit->Clear which can be assigned a shortcut but I can add a button also.

**Comment by Dreadeath** [comment]
Yes, I've already assigned it, but button is for mouse-only interaction.

**Comment by ludouzi** [comment]
Done. It can be enabled in `Settings->Playlist->Show clear button`.
